cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -100,7 +101,9 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
     driversPage.list().actionMenu('Azure AKS').getMenuItem('Deactivate').click();
     const deactivateDialog = new DeactivateDriverDialogPo();
 
-    deactivateDialog.deactivate();
+    deactivateDialog.deactivate().then(() => {
+      reenableAKS = true;
```

Comment:
Please, add an API call to ensure that the state of the container driver has successfully changed with the click() before changing `reenableAKS`